Packs/ConcentricAI/ReleaseNotes/1_2_0.md

```diff
@@ -0,0 +1,7 @@
+#### Integrations
+##### ConcentricAI
+- Added enhancement to playbook and overall functioning.
+#### Playbooks
+##### ConcentricAI Demo Playbook
+- The Playbook now enables one to fetch all file details, file sharing permissions
+     ,user details of the owner along with summaryReport of the incident.
```

Comment:
```suggestion
- The Playbook now enables one to fetch all file details, file sharing permissions, user details of the owner along with summaryReport of the incident.
```

I guess that the new line was made by mistake: 
![image](https://user-images.githubusercontent.com/50326704/110297956-aa43fa80-7ffc-11eb-9940-fd3b7f13e56a.png)